package.json

```diff
@@ -21,13 +21,12 @@
   },
   "homepage": "https://github.com/driftyco/ionic-module-template#readme",
   "devDependencies": {
```

Comment:
@larssn 

AFAIK these guides are meant for your main project, and not your "library" modules. I don't compile my modules with Webpack, I just compile them with `ngc` as es5/es6 modules and let Ionic's compiler consume them.

You can inline the CSS if you want, that will still work, I've done that [here](https://github.com/zyra/ionic-image-loader/blob/master/src/components/img-loader.ts#L8). In the link I provided earlier, I use SASS because the project is more complex, and at one point I was using Ionic SASS variables in there.